Formula/genext2fs.rb

```diff
@@ -0,0 +1,19 @@
+class Genext2fs < Formula
+  desc "Generates an ext2 filesystem as a normal (non-root) user"
+  homepage "http://freecode.com/projects/genext2fs"
+  url "https://downloads.sourceforge.net/projects/genext2fs/files/genext2fs/1.4.1/genext2fs-1.4.1.tar.gz"
+  sha256 "404dbbfa7a86a6c3de8225c8da254d026b17fd288e05cec4df2cc7e1f4feecfc"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/genext2fs", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!